docker/Dockerfile.rttools

```diff
@@ -28,43 +22,53 @@ echo 'PATH=${FSLDIR}/bin:${PATH}' >> ~/.bashrc && \
 echo 'export FSLDIR PATH' >> ~/.bashrc && \
 echo 'export LD_LIBRARY_PATH=${FSLDIR}/lib:${FSLDIR}/fslpython/lib:${LD_LIBRARY_PATH}' >> ~/.bashrc && \
 popd && \
-echo "## FSL Install Complete ##"
-
-
-# Install ANTs
-RUN \
-echo "## Install ANTs ##"  && \
+echo "## FSL Install Complete ##" && \
+# Remove bulk of FSL (inc. fslview!) leaving only binary analysis functions
+echo "## Removing non-essential FSL utilites! ##" && \
+source ~/.bashrc && \
+cd $FSLDIR && \
+rm -r config data doc extras fslpython include lib python refdoc src tcl  && \
+rm -r etc/default_flobs.flobs etc/flirtsch etc/js etc/luts etc/matlab && \
+rm build build.log && \
+rm ${FSLDIR}/bin/fslview* ${FSLDIR}/bin/eddy* && \
+echo "## Unbulking FSL Complete ##" && \
+# Install ANTs (reduced version with limited functions)
+echo "## Install ANTs ##" && \
 pushd ~/Downloads && \
 wget https://sourceforge.net/projects/advants/files/ANTS/ANTS_Latest/ANTs-2.1.0-rc3-Linux.tar.gz/download -O ANTs-2.1.0-rc3-Linux.tar.gz && \
 tar xzvf ANTs-2.1.0-rc3-Linux.tar.gz && \
 mv ANTs-2.1.0-Linux /opt/ && \
+cd /opt/ANTs-2.1.0-Linux/bin && \
+rm -r /opt/ANTs-2.1.0-Linux/lib && \
+ls -1 | egrep -v "^(ANTS|antsApplyTransforms|antsApplyTransformsToPoints|antsAlignOrigin|antsRegistration|antsTransformInfo)$" | xargs rm -r && \
 echo 'export PATH=${PATH}:/opt/ANTs-2.1.0-Linux/bin' >> ~/.bashrc && \
-rm ANTs-2.1.0-rc3-Linux.tar.gz && \
+rm ~/Downloads/ANTs-2.1.0-rc3-Linux.tar.gz && \
 popd && \
-echo "## ANTs Install Complete ##"
-
-# Install C3D tool
-RUN \
+echo "## ANTs Install Complete ##" && \
+# Install C3D tool (reduced version: only c3d and c3d_affine_tool)
 echo "## Install C3D ##" && \
 pushd ~/Downloads && \
 wget https://sourceforge.net/projects/c3d/files/c3d/Nightly/c3d-nightly-Linux-gcc64.tar.gz/download -O c3d-nightly-Linux-gcc64.tar.gz && \
 tar xzvf c3d-nightly-Linux-gcc64.tar.gz && \
-dirname=$(find . -maxdepth 1 -type d -name "c3d*") && \
-mv $dirname /opt/ && \
-c3dpath=$(find /opt -maxdepth 1 -type d -name "c3d*") && \
-echo 'export PATH=${PATH}:'"${c3dpath}/bin" >> ~/.bashrc && \
+mv c3d-1.3.0-Linux-gcc64 /opt/ && \
```

Comment:
Make generic for c3d version since we are downloading the latest version. See original rttools dockerfile.